.rubocop_todo.yml

```diff
@@ -27,7 +27,12 @@ Lint/NonDeterministicRequireOrder:
 # Offense count: 4
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 135
+  Max: 140
+
+# Offense count: 1
+RSpec/AnyInstance:
+  Exclude:
+    - 'spec/components/embargo_detail_component_spec.rb'
```

Comment:
This can go away... I disabled this in `.rubocop.yml` and probably got missed in a rebase.